crm_claim_rma/models/claim_line.py

```diff
@@ -0,0 +1,408 @@
+# -*- coding: utf-8 -*-
+# © 2015 Vauxoo
+# © 2013 Camptocamp
+# © 2009-2013 Akretion,
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import calendar
+import math
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from openerp import _, api, exceptions, fields, models
+from openerp.tools import (DEFAULT_SERVER_DATE_FORMAT,
+                           DEFAULT_SERVER_DATETIME_FORMAT)
+
+from .invoice_no_date import InvoiceNoDate
+from .product_no_supplier import ProductNoSupplier
+
+
+class ClaimLine(models.Model):
+
+    _name = "claim.line"
+
+    _inherit = 'mail.thread'
+    _description = "List of product to return"
+    _rec_name = "display_name"
+
+    SUBJECT_LIST = [('none', 'Not specified'),
+                    ('legal', 'Legal retractation'),
+                    ('cancellation', 'Order cancellation'),
+                    ('damaged', 'Damaged delivered product'),
+                    ('error', 'Shipping error'),
+                    ('exchange', 'Exchange request'),
+                    ('lost', 'Lost during transport'),
+                    ('perfect_conditions',
+                     'Perfect Conditions'),
+                    ('imperfection', 'Imperfection'),
+                    ('physical_damage_client',
+                     'Physical Damage by Client'),
+                    ('physical_damage_company',
+                     'Physical Damage by Company'),
+                    ('other', 'Other')]
+    WARRANT_COMMENT = [
+        ('valid', _("Valid")),
+        ('expired', _("Expired")),
+        ('not_define', _("Not Defined"))]
+
+    number = fields.Char(
+        readonly=True,
+        default='/',
+        help='Claim Line Identification Number')
+    company_id = fields.Many2one(
+        'res.company', string='Company', readonly=False,
+        change_default=True,
+        default=lambda self: self.env['res.company']._company_default_get(
+            'claim.line'))
+    date = fields.Date('Claim Line Date',
+                       select=True,
+                       default=fields.date.today())
+    name = fields.Char('Description', default='none', required=True,
+                       help="More precise description of the problem")
+    priority = fields.Selection([('0_not_define', 'Not Define'),
+                                 ('1_normal', 'Normal'),
+                                 ('2_high', 'High'),
+                                 ('3_very_high', 'Very High')],
+                                'Priority', default='0_not_define',
+                                compute='_set_priority',
```

Comment:
Should we take advantage of this PR to use `_compute_priority` [standard](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field)?